modules/RunAlmanack.nf

```diff
@@ -16,27 +16,27 @@
  * - repo_name: Repository name
  * - repo_dir: Path to cloned repository
  * - out_dir: Output directory
- * - status_repo.txt: Previous status file
+ * - status_file: Path to previous status file
  * 
  * Output: Tuple containing:
  * - repo_url: GitHub repository URL
  * - repo_name: Repository name
+ * - repo_dir: Path to cloned repository
  * - out_dir: Output directory
  * - status_almanack_<repo_name>.txt: Updated status file with Almanack results
+ * - almanack_results.json: JSON file with Almanack analysis results
  */
 
 process RunAlmanack {
     container 'python:3.11'
```

Comment:
Is there a container that you could use that has this tool installed already? It's not ideal to be installing things within a process